rmgpy/rmg/pdep.py

```diff
@@ -472,22 +472,49 @@ def remove_disconnected_reactions(self):
                 logging.info('Removing rxn: {}'.format(rxn))
                 self.pathReactions.remove(rxn)
                 
+        nrxns = []
+        for nrxn in self.netReactions:
+           prod = nrxn.products
+           if nrxn.products not in keptProducts or nrxn.reactants not in keptProducts:
```

Comment:
`nrxn.products` is a list of length 1 to 3, right? Can an `in` or `not in` check be performed that way? Shouldn't we iterate through the products in `nrxn.products` before the check?

Reply:
keptProducts is constructed a list of list of species (from a pdep network perspective this makes sense since each represents a configuration) so this should work fine.